datafusion/expr/src/operator.rs

```diff
@@ -218,37 +218,33 @@ impl Operator {
     }
 
     /// Get the operator precedence
-    /// use <https://www.postgresql.org/docs/7.0/operators.htm#AEN2026> as a reference
+    /// use <https://www.postgresql.org/docs/7.2/sql-precedence.html> as a reference
     pub fn precedence(&self) -> u8 {
```

Comment:
It looks like this is only used in a few places for formatting the parenthesis in a `Display` trait on `BinaryExpr`. I don't think that will have any major impact on library users.